glotaran/builtin/megacomplexes/coherent_artifact/coherent_artifact_megacomplex.py

```diff
@@ -84,6 +85,7 @@ def finalize_data(
                 (global_dimension, "coherent_artifact_order"),
                 dataset.clp.sel(clp_label=self.compartments()).values,
             )
+        retrieve_irf(dataset_model, dataset, dataset_model.get_global_dimension())
```

Comment:
Why is this at a different indentation level as the rest of the code inside the `if not is_full_model:` block. 
It appears as if the code was written in such a way that only if `not is_full_model` is true things are added to the result, so it's not clear what the intention is now. 

Please correct indentation or clarify in a comment.

Reply:
To me, it looks like that if an IRF is present its data will always be added to the result no matter if it is a full model or not (if not already added before).

Reply:
This is simply that I removed the check for if the irf has already been done to the retrieve irf util so the coherent artifact can import and use it without me geting a banghammer for duplicatioin